src/Analysis/Codelyzer.Analysis.CSharp/Handlers/InvocationExpressionHandler.cs

```diff
@@ -41,12 +38,13 @@ private void SetMetaData(InvocationExpressionSyntax syntaxNode)
 
             foreach (var argumentSyntax in syntaxNode.ArgumentList.Arguments)
             {
-                Parameter parameter = new Parameter();
-                if (argumentSyntax.Expression != null)
-                    parameter.Name = argumentSyntax.Expression.ToString();
+                var argument = new Argument
+                {
+                    Identifier = argumentSyntax.Expression.ToString(),
+                    SemanticType = SemanticHelper.GetSemanticType(argumentSyntax.Expression, SemanticModel)
+                };
 
-                parameter.SemanticType =
-                    SemanticHelper.GetSemanticType(argumentSyntax.Expression, SemanticModel);
+                Model.Arguments.Add(argument);
```

Comment:
This might be a breaking change for other applications. We need to keep the parameter and add the argument.